### ISSUE TITLE
HTML DOCTYPE required for the library to work

### ISSUE THREAD

**Issue by themaxsandelin** [darkroomengineering/lenis#420]
**Describe the bug**
If you determine this to be a "bug" or not, is up to you, but I just spent 2 hours trying to get this library to work on my site and used various online tools like Codepen and Codesandbox to see if it was some issue with the versioning of the library. It turned out that I simply had forgot to add the `<!DOCTYPE html>` at the top of my HTML document.

I don't really mind if this is required or not, there might be a reason. However, it would be very helpful if you at least outline this in the documentation. I know the doctype is standard, however it's very easy to forget. :)

**To Reproduce**
Simply set up a basic HTML website and try using the library, and omit the `<!DOCTYPE html>` at the top of the page.

**Comment by maurodf0** [comment]
Hi @themaxsandelin, thanks for pointing out this one.
Had this problem and now works fine.

Only the scroll in 'inspect' mode worked cause trigged the 'native' scroll

**Comment by clementroche** [comment]
Hello @themaxsandelin this has been fixed in [v1.1.19-dev.0](https://www.npmjs.com/package/lenis/v/1.1.19-dev.0), can you confirm?

**Comment by themaxsandelin** [comment]
@clementroche thanks for looking into this! I quickly created a test in Codesandbox with React and added v1.1.19-dev.0 as a dependency, but the issue seems to remain? You can find the sandbox link below:

https://codesandbox.io/p/sandbox/h4kt4q

**Comment by clementroche** [comment]
Indeed, please try [v1.1.19-dev.5](https://www.npmjs.com/package/lenis/v/1.1.19-dev.5)

https://codesandbox.io/p/sandbox/lenis-test-forked-mkry9t

**Comment by themaxsandelin** [comment]
This version works! Nicely done 👌
